{
    "active_lock_reason": null,
    "assignee": null,
    "assignees": [],
    "author_association": "NONE",
    "body": "### Problem\n\nWhile editing a file, Neovim segfaulted. I don't know what caused it, and I don't know how to reproduce it, but I've attached the traceback from the core dump if it helps. \r\n\r\n```\r\nUsing host libthread_db library \"/lib/x86_64-linux-gnu/libthread_db.so.1\".\r\nCore was generated by `nvim --embed xdg-user-dir'.\r\nProgram terminated with signal SIGSEGV, Segmentation fault.\r\n#0  0x00007f83e86753af in ?? ()\r\n[Current thread is 1 (Thread 0x7f83dc0cb740 (LWP 95296))]\r\n(gdb) bt full\r\n#0  0x00007f83e86753af in ?? ()\r\nNo symbol table info available.\r\n#1  0x0000000041d9dda0 in ?? ()\r\nNo symbol table info available.\r\n#2  0x00007f83dc1e6921 in ?? () from /lib/x86_64-linux-gnu/libluajit-5.1.so.2\r\nNo symbol table info available.\r\n#3  0x00007f83dc1f5f13 in lua_pcall () from /lib/x86_64-linux-gnu/libluajit-5.1.so.2\r\nNo symbol table info available.\r\n#4  0x00005632542637bc in nlua_pcall ()\r\nNo symbol table info available.\r\n#5  0x000056325426884b in nlua_call_ref ()\r\nNo symbol table info available.\r\n#6  0x00005632541aa621 in ?? ()\r\nNo symbol table info available.\r\n#7  0x00005632541ab0ad in decor_providers_invoke_end ()\r\nNo symbol table info available.\r\n#8  0x00005632541bde16 in update_screen ()\r\nNo symbol table info available.\r\n#9  0x00005632541be0ef in ins_redraw ()\r\nNo symbol table info available.\r\n#10 0x00005632541c0f7d in ?? ()\r\nNo symbol table info available.\r\n#11 0x000056325434c955 in state_enter ()\r\nNo symbol table info available.\r\n#12 0x00005632541c25ff in ?? ()\r\nNo symbol table info available.\r\n#13 0x00005632541c27ec in edit ()\r\nNo symbol table info available.\r\n#14 0x00005632542b64d0 in ?? ()\r\nNo symbol table info available.\r\n#15 0x00005632542b6792 in ?? ()\r\nNo symbol table info available.\r\n#16 0x00005632542b3597 in ?? ()\r\nNo symbol table info available.\r\n#17 0x000056325434c985 in state_enter ()\r\nNo symbol table info available.\r\n#18 0x00005632542abbca in normal_enter ()\r\nNo symbol table info available.\r\n#19 0x0000563254136187 in main ()\r\nNo symbol table info available.\r\n```\n\n### Steps to reproduce\n\n:man_shrugging: \n\n### Expected behavior\n\nNo segfault\n\n### Neovim version (nvim -v)\n\nNVIM v0.10.0-dev\n\n### Vim (not Nvim) behaves the same?\n\nN/A\n\n### Operating system/version\n\nUbuntu 22.04.3 LTS\n\n### Terminal name/version\n\nwezterm 20230712-072601-f4abf8fd\n\n### $TERM environment variable\n\nwezterm\n\n### Installation\n\nUbuntu Neovim Nightly PPA (https://launchpad.net/~neovim-ppa/+archive/ubuntu/unstable)",
    "closed_at": null,
    "comment_data": [
        {
            "author_association": "NONE",
            "body": "As I have no reproduction instructions, and it could (possibly) be one of my plugins that caused this, feel free to close this issue if you feel there's no way to track down the original issue.",
            "created_at": "2023-08-11T06:37:53Z",
            "html_url": "https://github.com/neovim/neovim/issues/24658#issuecomment-1674273121",
            "id": 1674273121,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/24658",
            "node_id": "IC_kwDOAPphoM5jy2Fh",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/1674273121/reactions"
            },
            "updated_at": "2023-08-11T06:37:53Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/1674273121",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/117116764?v=4",
                "events_url": "https://api.github.com/users/aarondill/events{/privacy}",
                "followers_url": "https://api.github.com/users/aarondill/followers",
                "following_url": "https://api.github.com/users/aarondill/following{/other_user}",
                "gists_url": "https://api.github.com/users/aarondill/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/aarondill",
                "id": 117116764,
                "login": "aarondill",
                "node_id": "U_kgDOBvsPXA",
                "organizations_url": "https://api.github.com/users/aarondill/orgs",
                "received_events_url": "https://api.github.com/users/aarondill/received_events",
                "repos_url": "https://api.github.com/users/aarondill/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/aarondill/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/aarondill/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/aarondill"
            }
        },
        {
            "author_association": "MEMBER",
            "body": "This may be a problem with the LuaJIT version used by the package. Can you try nightly appimage instead and seem if that also crashes?",
            "created_at": "2023-08-11T06:42:22Z",
            "html_url": "https://github.com/neovim/neovim/issues/24658#issuecomment-1674276457",
            "id": 1674276457,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/24658",
            "node_id": "IC_kwDOAPphoM5jy25p",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/1674276457/reactions"
            },
            "updated_at": "2023-08-11T06:42:22Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/1674276457",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/35768171?v=4",
                "events_url": "https://api.github.com/users/zeertzjq/events{/privacy}",
                "followers_url": "https://api.github.com/users/zeertzjq/followers",
                "following_url": "https://api.github.com/users/zeertzjq/following{/other_user}",
                "gists_url": "https://api.github.com/users/zeertzjq/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/zeertzjq",
                "id": 35768171,
                "login": "zeertzjq",
                "node_id": "MDQ6VXNlcjM1NzY4MTcx",
                "organizations_url": "https://api.github.com/users/zeertzjq/orgs",
                "received_events_url": "https://api.github.com/users/zeertzjq/received_events",
                "repos_url": "https://api.github.com/users/zeertzjq/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/zeertzjq/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/zeertzjq/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/zeertzjq"
            }
        },
        {
            "author_association": "NONE",
            "body": "Sure, I'd be happy to do so, but, just as a friendly reminder, I still have no way to intentionally reproduce this crash, so the only way I'd know if it worked or not is if I got another segfault with the same backtrace.\r\n\r\nI'm sorry I can't be of more help, but I was just editing a file as normal when it occurred, and reopening the file does not result in another crash.",
            "created_at": "2023-08-11T07:21:57Z",
            "html_url": "https://github.com/neovim/neovim/issues/24658#issuecomment-1674313538",
            "id": 1674313538,
            "issue_url": "https://api.github.com/repos/neovim/neovim/issues/24658",
            "node_id": "IC_kwDOAPphoM5jy_9C",
            "performed_via_github_app": null,
            "reactions": {
                "+1": 0,
                "-1": 0,
                "confused": 0,
                "eyes": 0,
                "heart": 0,
                "hooray": 0,
                "laugh": 0,
                "rocket": 0,
                "total_count": 0,
                "url": "https://api.github.com/repos/neovim/neovim/issues/comments/1674313538/reactions"
            },
            "updated_at": "2023-08-11T07:21:57Z",
            "url": "https://api.github.com/repos/neovim/neovim/issues/comments/1674313538",
            "user": {
                "avatar_url": "https://avatars.githubusercontent.com/u/117116764?v=4",
                "events_url": "https://api.github.com/users/aarondill/events{/privacy}",
                "followers_url": "https://api.github.com/users/aarondill/followers",
                "following_url": "https://api.github.com/users/aarondill/following{/other_user}",
                "gists_url": "https://api.github.com/users/aarondill/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/aarondill",
                "id": 117116764,
                "login": "aarondill",
                "node_id": "U_kgDOBvsPXA",
                "organizations_url": "https://api.github.com/users/aarondill/orgs",
                "received_events_url": "https://api.github.com/users/aarondill/received_events",
                "repos_url": "https://api.github.com/users/aarondill/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/aarondill/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/aarondill/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/aarondill"
            }
        }
    ],
    "comments": 3,
    "comments_url": "https://api.github.com/repos/neovim/neovim/issues/24658/comments",
    "created_at": "2023-08-11T06:36:45Z",
    "events_url": "https://api.github.com/repos/neovim/neovim/issues/24658/events",
    "html_url": "https://github.com/neovim/neovim/issues/24658",
    "id": 1846274424,
    "labels": [
        {
            "color": "FBCA04",
            "default": false,
            "description": "waiting for reply from the author",
            "id": 248229439,
            "name": "needs:response",
            "node_id": "MDU6TGFiZWwyNDgyMjk0Mzk=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/needs:response"
        },
        {
            "color": "F9D0C4",
            "default": false,
            "description": "issue reporting a crash or segfault",
            "id": 435854234,
            "name": "bug-crash",
            "node_id": "MDU6TGFiZWw0MzU4NTQyMzQ=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/bug-crash"
        },
        {
            "color": "c5def5",
            "default": false,
            "description": null,
            "id": 573222693,
            "name": "lua",
            "node_id": "MDU6TGFiZWw1NzMyMjI2OTM=",
            "url": "https://api.github.com/repos/neovim/neovim/labels/lua"
        }
    ],
    "labels_url": "https://api.github.com/repos/neovim/neovim/issues/24658/labels{/name}",
    "locked": false,
    "milestone": null,
    "node_id": "I_kwDOAPphoM5uC-l4",
    "number": 24658,
    "performed_via_github_app": null,
    "reactions": {
        "+1": 0,
        "-1": 0,
        "confused": 0,
        "eyes": 0,
        "heart": 0,
        "hooray": 0,
        "laugh": 0,
        "rocket": 0,
        "total_count": 0,
        "url": "https://api.github.com/repos/neovim/neovim/issues/24658/reactions"
    },
    "repository_url": "https://api.github.com/repos/neovim/neovim",
    "state": "open",
    "state_reason": null,
    "timeline_url": "https://api.github.com/repos/neovim/neovim/issues/24658/timeline",
    "title": "Segfault in libluajit",
    "updated_at": "2023-08-11T11:40:04Z",
    "url": "https://api.github.com/repos/neovim/neovim/issues/24658",
    "user": {
        "avatar_url": "https://avatars.githubusercontent.com/u/117116764?v=4",
        "events_url": "https://api.github.com/users/aarondill/events{/privacy}",
        "followers_url": "https://api.github.com/users/aarondill/followers",
        "following_url": "https://api.github.com/users/aarondill/following{/other_user}",
        "gists_url": "https://api.github.com/users/aarondill/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/aarondill",
        "id": 117116764,
        "login": "aarondill",
        "node_id": "U_kgDOBvsPXA",
        "organizations_url": "https://api.github.com/users/aarondill/orgs",
        "received_events_url": "https://api.github.com/users/aarondill/received_events",
        "repos_url": "https://api.github.com/users/aarondill/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/aarondill/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/aarondill/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/aarondill"
    }
}